src/translate.zig

```diff
@@ -18,14 +18,15 @@ pub fn register_function(
     }
 }
 
-pub fn throw(env: c.napi_env, comptime message: [:0]const u8) error{ExceptionThrown} {
+const TranslationErrors = error{ExceptionThrown};
```

Comment:
I think in the Zig std lib etc. they just use the singular for the name of the error set type, e.g. here we could do `TranslationError`.

Reply:
Thanks for making these error sets explicit, rather than inferred.